src/index.html

```diff
@@ -2,13 +2,66 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name = "viewport"
+          content="width=device-width, initial-scale=1.0">
```

Comment:
You should remember about code style.
If we have 3 or more attributes, they should be on a new line
```suggestion
    <meta
      name = "viewport"
      content="width=device-width,
      initial-scale=1.0"
   >
```